src/runtime/composables-v5/useStrapi.ts

```diff
@@ -61,9 +62,10 @@ export const useStrapi = <T>(): StrapiV5Client<T> => {
    * @param  {string} contentType - Content type's name pluralized
    * @param  {string} documentId - ID of entry to be updated
    * @param  {Record<string, any>} data - Form data
+   * @param  {Strapi5RequestParams} [params] - Query parameters
    * @returns Promise<T>
    */
-  const update = <T>(contentType: string, documentId: string | Partial<T>, data?: Partial<T>): Promise<Strapi5ResponseSingle<T>> => {
+  const update = <T>(contentType: string, documentId: string | Partial<T>, data?: Partial<T>, params: Strapi5RequestParams = {}): Promise<Strapi5ResponseSingle<T>> => {
```

Comment:
U probably wanted to assign the params to the client. In that case u'd fix pipelines.
` return client(path, { method: 'PUT', body: { data }, params })`

Reply:
Ty. Idk how i missed that lol